### DISCUSSION TITLE
Keyboard Navigation for Video Player (no control focus)

### DISCUSSION THREAD

**Discussion by Ristophonics** [fancyapps/ui#107] [Q&A]
Using Fancyboxv3 to display a video, it had default keyboard navigation of the video controls.

Trying to implement the same thing in V4 but the ability to tab navigate to video controls is .. not the same.
Focus is hard to track with just the keyboard nav..

EDIT

Example: https://aecomviz.com/fancyhelp/

How to Reproduce Issue,  
1) Tab Navigate to the Welcome link
2) Hit Enter (opens fancybox)
3) Try to Tab to the Video controls

note: with Toolbar: { autoEnable:false }   
Tab navigation can access the toolbar button but not the video player controls.
For the Video to get focus() from Tab navigation the user has to use "shift tab" to get back to the video player element. The order of focus() looks like it is in reverse... close button is first.. then full screen.. then the video element.. hope that helps 

Thank you!!!

**Comment by fancyapps** [comment]
Hi,

Please, update to the latest version. Focus management has been rewritten completely and it should work better. 

Anyway, There's no native focus trap provided by browser and this is kind of a hacky solution.
